README.md

```diff
@@ -410,4 +426,221 @@ See documentation for [django-environment-config] for more details.
 
 ## Frontend
 
-TODO: Check [old readme](README-ui.md)
+### Codegen
+
+Needs to be run if either the backend schema or any frontend query changes.
```

Comment:
First, explain briefly what codegen is and why we use it: "To generate types for GraphGL operations, frontend uses a codegen tool..." etc.